src/main/resources/assets/farmersdelight/lang/ko_kr.json

```diff
@@ -10,6 +10,7 @@
   "block.farmersdelight.jungle_cabinet": "정글나무 식료품 선반",
   "block.farmersdelight.acacia_cabinet": "아카시아 식료품 선반",
   "block.farmersdelight.dark_oak_cabinet": "짙은 참나무 식료품 선반",
+  "block.farmersdelight.mangrove_cabinet": "맹그로브 식료품 ",
```

Comment:
Thanks again for the update! Just a quick note:

This lang seems to be missing a word, compared to the other cabinet langs. Google Translate calls it "Mangrove food". Is this intended?